### PR TITLE
Initial site build

### PR DESCRIPTION
**Pull request by yghoor**
### Built bootstrap version of a portfolio site done according to spec from /curriculum-html-css/bootstrap-exercise.md

Figma template - https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471

### CODE REVIEW

**Comment by Cerdtruir** [comment]
Looks, good.

Im going to list the issues as i see them. 

The modal Github icon doesn't load.

I think the top portfolio item is supposed to change when scrolling over the lower ones in the grid. But thats not a big issue.

also the text over the grid portfolio items should change on hover. 

The form isnt expanding with the page like the figma template shows.

the form text box background should be grey

The button colors arent changing on hover and click



 But if there's no way to do some of these on bootstrap i dont think it matters.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,833 @@
...
+        </aside>
+  
+        <section class="col-11 d-flex flex-column pt-5">
+          <h1 class="text-warning display-3 fw-normal mt-5">Hey There.</h1> 
+
+          <h2 class="text-warning display-3 fw-normal">I'm Yahya</h2>
+  
+          <h2 class="text-white fw-normal">I am a Software Developer</h2>
+  
+          <p class="col-9 col-sm-7 col-md-6 col-lg-5 col-xl-4 col-xxl-3 text-white">
```
Comment (Cerdtruir) [inline index.html:94]: It seems like you have lots of column changes, could you have used a percentage or something else?

Comment (Cerdtruir) [reply index.html:94]: There are a few others places i saw you did the same thing

**Review by Cerdtruir — CHANGES_REQUESTED** [review]
Please have a look at the other comments and fix the main issues. Otherwise it looks good

there also these error when loading the page: 

modal-icon-export.svg:1          Failed to load resource: the server responded with a status of 404 ()
modal-icon-github.svg:1          Failed to load resource: the server responded with a status of 404 ()
/favicon.ico:1          Failed to load resource: the server responded with a status of 404 ()

**Comment by yghoor** [comment]
Necessary changes made

**Review by Cerdtruir — APPROVED** [review]

**Pull request merged**